packages/backend/package.json

```diff
@@ -43,6 +43,7 @@
   },
   "devDependencies": {
     "concurrently": "6.1.0",
+    "nock": "^13.1.0",
```

Comment:
We're using [exact versions](https://github.com/MetaFam/TheGame/commit/51089ae49e8ee2a7a41936fb22b27c2bbe77aae0). *(I'm uncertain as to the effectiveness of doing so, but that is the current standard.)*

Reply:
Good catch, thanks